openslides_backend/action/actions/user/user_mixin.py

```diff
@@ -173,3 +173,55 @@ def get_history_information(self) -> Optional[HistoryInformation]:
                     fqid_from_collection_and_id("user", instance["id"])
                 ] = instance_information
         return information
+
+
+class DuplicateCheckMixin(Action):
+    def init_duplicate_set(
+        self, usernames: List[str], names_and_emails: List[Any]
+    ) -> None:
+        # we don't want to call "get_all", so need to construct this large
+        # filter here.
+        filter_ = Or(
+            *[FilterOperator("username", "=", un) for un in usernames],
+            *[
+                And(
+                    FilterOperator("first_name", "=", entry[0]),
+                    FilterOperator("last_name", "=", entry[1]),
+                    FilterOperator("email", "=", entry[2]),
```

Comment:
The `search_users` presenter matches case-insensitive. For the import I could see that we do not want this, but this would mean the matching algorithm differs between the presenter and the import. @rrenkert what do you think?

Reply:
I use the `search_users` presenter to get the possible duplicates.